app/models/pipeline/static_analysis.py

```diff
@@ -81,18 +81,9 @@ async def review(self, mr: MergeRequest) -> PipelineResult:
 
         # 创建评论，每个文件最多保留2条最重要的评论
         comments = []
-        file_comment_count = {}
         for ai_comment in ai_review.comments:
             if ai_comment.type == "praise":
```

Comment:
[From AI Code Reviewer][bot] Similar to the `logic_review.py`, consider reinstating the logic to limit comments per file to improve review focus.